packages/http/src/mocker/generator/HttpParamGenerator.ts

```diff
@@ -0,0 +1,64 @@
+import { IHttpOperation, IHttpParam, IHttpPathParam } from '@stoplight/types';
+import { INodeExample, INodeExternalExample } from '@stoplight/types';
+import * as Array from 'fp-ts/lib/Array';
+import * as Option from 'fp-ts/lib/Option';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { JSONSchema } from '../../types';
+import { generate as generateDynamicExample } from './JSONSchema';
+
+function improveSchema(schema: JSONSchema) {
+  const newSchema = { ...schema };
```

Comment:
Maybe let's use deepClone by lodash?

Reply:
Actually it is cloned anyway by this recurrence so cloning it again would be showing no respect for CPU cycles.

Reply:
I was suggesting to use clone instead of the ... but your solution is just ok as it is!